project/plugins.sbt

```diff
@@ -8,11 +8,11 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
 
 ThisBuild / libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
 
-addSbtPlugin("uk.gov.hmrc"         % "sbt-auto-build"           % "3.9.0")
+addSbtPlugin("uk.gov.hmrc"         % "sbt-auto-build"           % "3.11.0")
```

Comment:
This is two versions behind. Please bump to [3.13.0](https://github.com/hmrc/sbt-auto-build/releases/tag/v3.13.0), thanks.